src/Impostor.Server/Net/Inner/Objects/InnerPlayerControl.cs

```diff
@@ -273,6 +275,7 @@ public override async ValueTask HandleRpc(ClientPlayer sender, ClientPlayer? tar
                         player.Die(DeathReason.Kill);
                     }
 
+                    await _eventManager.CallAsync(new PlayerMurderEvent(_game, sender, this, player));
```

Comment:
Please put this into the if statement above.

Reply:
1ad1a64a08250393ba51fc4771b7d238bc64aba9